app/components/shared/restructured_work_history/review_component.rb

```diff
@@ -56,5 +56,9 @@ def no_work_experience_value
     def breaks_in_work_history?
       CheckBreaksInWorkHistory.call(@application_form)
     end
+
+    def change_path
+      CandidateInterface::ApplicationFormPresenter.new(@application_form).work_experience_path('return-to' => 'application-review') if @return_to_application_review
```

Comment:
What will this do from the user's POV if `@return_to_application_review` isn't set?

Reply:
If we only see the nested incomplete component from the application review page, do we need the if `@return_to_application_review` conditional? We would always want to return to the application review page

Reply:
I think in this case better to be consistent (with the other `ReviewComponent`s) than to be right? We should probably review that lot anyway to see if that arg could be removed across the board... or whether they could be reused in places that arg would be false.